helm/charts.go

```diff
@@ -0,0 +1,31 @@
+package helm
+
+import (
+	"path/filepath"
+
+	"github.com/google/go-github/github"
+)
+
+// GetWorkflowChart returns a helmChart that represents the workflow chart and
+// the files in it that need updating for a release
+func GetWorkflowChart(ghClient *github.Client) (*Chart, error) {
+	return newChart(ghClient, "workflow-dev", []string{
+		"README.md",
+		"Chart.yaml",
+	})
+}
+
```

Comment:
this and `GetRouterChart` could use godoc descriptions like `GetWorkflowChart` above